include/zephyr/net/wifi.h

```diff
@@ -108,12 +108,22 @@ enum wifi_eap_type {
 	WIFI_EAP_TYPE_MSCHAPV2 = 26,
 };
 
-/** @brief Enterprise security WPA3 suiteb types. */
-enum wifi_suiteb_type {
-	/** suiteb. */
-	WIFI_SUITEB = 1,
-	/** suiteb-192. */
-	WIFI_SUITEB_192,
+/** @brief WPA3 Enterprise security types. */
+enum wifi_wpa3_enterprise_type {
+	/** no wpa3 enterprise. */
+	WIFI_WPA3_ENTERPRISE_NONE = 0,
+	/** wpa3 enterprise suiteb. */
+	WIFI_WPA3_ENTERPRISE_SUITEB = 1,
+	/** wpa3 enterprise suiteb-192. */
+	WIFI_WPA3_ENTERPRISE_SUITEB_192,
+	/** wpa3 enterprise only. */
```

Comment:
`ONLY` is ambiguous, IIUC, this means WPA3 Enterprise but without Suite-B? Is that even supported?